makefile

```diff
@@ -137,7 +137,7 @@ BUILD_DIR := $(MFEM_BUILD_DIR)
 BUILD_REAL_DIR := $(abspath $(BUILD_DIR))
 ifneq ($(BUILD_REAL_DIR),$(MFEM_REAL_DIR))
    BUILD_SUBDIRS = $(DIRS) config $(EM_DIRS) doc $(TEST_DIRS)
-   CONFIG_FILE_DEF = -DMFEM_CONFIG_FILE='"$(BUILD_REAL_DIR)/config/_config.hpp"'
+   CONFIG_FILE_DEF = -DMFEM_CONFIG_FILE='\"$(BUILD_REAL_DIR)/config/_config.hpp\"'
```

Comment:
This seems to break the out-of-source build for me, e.g. I get errors when I try this:
```console
cd mfem
mkdir build
cd build
make -f ../makefile config
make
```

Reply:
You're right, I'll look on the CEED benchmark side why I needed that there.

Reply:
Did this get taken care of?

Reply:
Yes, see https://github.com/mfem/mfem/pull/1198/commits/fde20508f165cbb256fd9ce752229909b8572c4f